util/log/log.go

```diff
@@ -0,0 +1,28 @@
+package log
+
+import (
+	"context"
+
+	"github.com/containerd/containerd/log"
+	"github.com/sirupsen/logrus"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func init() {
+	log.G = GetLogger
```

Comment:
for a simple change to support buildkit's logs move this line out of util to `cmd/buildkitd` and then you can replace the logrus calls with `util/log.GetLogger()` without containerd/log dependency.

Can do this in separate PR as well.

Reply:
And when you do that, don't use `log` package name as it collides with stdlib in imports. `bklog`or `logger.Get()` or smth else that doesn't collide is better.